app/ui/src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts

```diff
@@ -66,7 +66,8 @@ const MAPPING_KEY = 'atlasmapping';
 })
 export class DataMapperHostComponent extends FlowPage implements OnInit {
   routeSubscription: Subscription;
-  initialized = false;
+  initializedCountDown = 1;
```

Comment:
Maybe name this `outstandingTasks` instead?  That might make it more clear what this does when we have to go back into this file.